templates/reports/cqm/cqm-form.html.twig

```diff
@@ -0,0 +1,153 @@
+                                    <select {{ dis_text }} id='form_plan_filter' name='form_plan_filter' class='form-control'>
+                                        {% for plan in plans %}
+                                            <option value="{{ plan.value|attr }}" {% if plan.selected %}selected{% endif %}>
+                                                {{ plan.label|text }}
+                                            </option>
+                                        {% endfor %}
+                                    </select>
+                                </td>
+                            </tr>
+                            {% endif %}
+                            <tr>
+                                <td class='col-form-label'>
+                                    {{ 'Provider'|xlt  }}:
+                                </td>
+                                <td>
+                                    <select {{ dis_text }} id='form_provider' name='form_provider' class='form-control'>
+                                        {% for provider in providerReportOptions %}
+                                            <option value="{{ provider.value|attr }}" {% if provider.selected %}selected{% endif %}>
+                                                {{ provider.label|text }}
+                                            </option>
+                                        {% endfor %}
+                                    </select>
+                                </td>
+                            </tr>
+
+                            <tr>
+                                <td class='col-form-label'>
+                                    {{ 'Provider Relationship'|xlt  }}:
+                                </td>
+                                <td>
+                                    <select {{ dis_text }} id='form_pat_prov_rel' name='form_pat_prov_rel' class='form-control'>
+                                        {% for relationship in providerRelationship %}
+                                            <option value="{{ relationship.value|attr }}" {% if relationship.selected %}selected{% endif %}>
+                                                {{ relationship.label|text }}
+                                            </option>
+                                        {% endfor %}
+                                    </select>
+                                </td>
+                            </tr>
+                            {% if show_manual_labs %}
+                            <tr>
+                                <td>
+                                    {{ 'Number labs'|xlt }}:<br />
+                                    ({{ 'Non-electronic'|xlt }})
+                                </td>
+                                <td>
+                                    <input {{ dis_text|text }} type="text" id="labs_manual_entry" name="labs_manual_entry" class='form-control' value="{{ labs_manual|attr }}">
```

Comment:
will escaping dis_text break this? (looks like it has quotes in it that are likely needed for html and is not escaped above)

Reply:
As far as I can tell escaping it didn't break it.